dapps/marketplace/src/pages/growth/Welcome.js

```diff
@@ -239,7 +239,7 @@ class GrowthWelcome extends Component {
             </div>
           </div>
           <div
-            className={`${isMobile ? 'd-none' : ''} origin-showcase col-10`}
+            className={`${isMobile ? 'd-none' : ''} origin-showcase col-8`}
```

Comment:
Just a thought. We can probably remove the `${isMobile ? 'd-none' : ''}` and do it with CSS media queries. That doesn't have to be in JS if we are just hiding it.

May be we can also just skip rendering the element on Mobile.

Reply:
Yup not rendering on mobile is a great idea, changing to that